cmd/influx_inspect/export.go

```diff
@@ -99,19 +106,25 @@ func (c *cmdExport) writeFiles() error {
 		w = gzip.NewWriter(w)
 	}
 
-	// Write out all the DDL
-	fmt.Fprintln(w, "# DDL")
-	for key, _ := range c.files {
-		keys := strings.Split(key, string(byte(os.PathSeparator)))
-		fmt.Fprintf(w, "CREATE DATABASE %s\n", keys[0])
-		fmt.Fprintf(w, "CREATE RETENTION POLICY %s ON %s DURATION inf REPLICATION 1\n", keys[1], keys[0])
+	if c.tsmDir == "" {
```

Comment:
Good point.  Maybe a followup change might be to add a `newDb` and `newRP` flag to allow generating the DDL?  Don't think that is necessary for this PR though.